diff --git a/Archive/Tar.php b/Archive/Tar.php
index 10ac845..570b39c 100644
--- a/Archive/Tar.php
+++ b/Archive/Tar.php
@@ -44,6 +44,27 @@
 define('ARCHIVE_TAR_ATT_SEPARATOR', 90001);
 define('ARCHIVE_TAR_END_BLOCK', pack("a512", ''));
 
+if (!function_exists('gzopen') && function_exists('gzopen64')) {
+    function gzopen($filename, $mode, $use_include_path = 0)
+    {
+        gzopen64($filename, $mode, $use_include_path);
+    }
+}
+
+if (!function_exists('gztell') && function_exists('gztell64')) {
+    function gztell($zp)
+    {
+        gztell64($zp);
+    }
+}
+
+if (!function_exists('gzseek') && function_exists('gzseek64')) {
+    function gzseek($zp, $offset, $whence = SEEK_SET)
+    {
+        gzseek64($zp, $offset, $whence);
+    }
+}
+
 /**
  * Creates a (compressed) Tar archive
  *
